src/main/resources/db/migration/V1.0.0__create_songs_table.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS songs(
+ id INTEGER NOT NULL GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+ artistName VARCHAR(255) NOT NULL,
+ trackName VARCHAR(255) NOT NULL,
+ trackTime TIME NOT NULL,
+ genre VARCHAR (255) NOT NULL,
```

Comment:
Czy chcemy przechowywać gatunek jako tekstową wartość razem z utworem? Czy zawsze mamy pewność, że piosenka należy do jednego i tylko jednego gatunku? Dobra praktyka raczej kazałaby mieć listę gatunków gdzieś osobno.

Myślę, że to jest coś nad czym warto się zastanowić i to dość poważnie, ale niekoniecznie od razu teraz 😉 spokojnie można dodać kolumnę później.